scripts/run-e2e-config.ts

```diff
@@ -159,6 +159,7 @@ export const sfcVue: Parameters = {
   generator: fromDeps('vue', 'vue-loader', 'vue-template-compiler'),
   additionalDeps: [
     'webpack@webpack-4',
+    'react',
```

Comment:
While this is what is required I'm assuming you don't actually want Vue users to have to install React. Perhaps Storybook should set an alias for `react` to make sure it's resolved from the core?

@gaetanmaisse seems to have "regressed" in https://github.com/storybookjs/storybook/pull/12972

Reply:
It's weird as in the stated PR I added react to the deps of `@storybook/vue`: https://github.com/storybookjs/storybook/pull/12972/files#diff-3133f51bc714c0aa1f09aa806d0b57b4727c48342f4c570ffe531942fa17e074R42

So we shouldn't need to add `react` as a direct dep to make everything work together 🤔

Reply:
Hmm pb comes from ` @storybook/addon-docs` and not `@storybook/vue`:
> ERR! Module not found: Error: @storybook/addon-docs tried to access react (a peer dependency) but it isn't provided by your application; this makes the require call ambiguous and unsound.
ERR! 
ERR! Required package: react (via "react")

Reply:
So we are back to the issue we discussed a week ago with other SB foxes: https://github.com/storybookjs/storybook/pull/12972#issuecomment-720423808

@merceyz As a Yarn 2 master, do you have any idea?

Reply:
> Perhaps Storybook should set an alias for react to make sure it's resolved from the core? 

Are you speaking of a webpack alias? Cause I guess we already have one to be sure to use a single version of react, @ndelangen can confirm. However, it doesn't solve our _missing_ (peer)dependency issues 😞

Reply:
> do you have any idea?

Either dependency injection or have `@storybook/vue` export `react` so that the vue part of the docs would import it from there.

> Are you speaking of a webpack alias? 

Yes

> However, it doesn't solve our missing (peer)dependency issues disappointed

That is true, if none of the dependencies the docs has needs react, importing react from `@storybook/vue` seems to be the best option